### PR TITLE
CONCF-504 Trending Articles

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/CONCF-504

App part: https://github.com/Wikia/app/pull/7160

@rogatty @Warkot @hakubo

### CODE REVIEW

File: server/lib/MediaWiki.ts [hunk 1]
```diff
@@ -99,15 +99,16 @@ export class ArticleRequest {
...
-				title: title
+				title: title,
+				action: 'purge'
 			},
 			url: string;
 		if (redirect) {
 			urlParams.redirect = redirect;
 		}
-		url = createUrl(this.wikiDomain, 'wikia.php', urlParams);
+		url = createUrl('sandbox-mercury.elderscrolls.wikia.com', 'wikia.php', urlParams);
```
Comment (hakubo) [inline server/lib/MediaWiki.ts]: ups :)

Comment (dianafa) [inline server/lib/MediaWiki.ts]: well, its the only way to test it

**Comment by bognix** [comment]
This pull request should be created against curated-content branch not dev.

**Comment by rogatty** [comment]
Done properly in https://github.com/Wikia/mercury/pull/764.

**Pull request closed without merging**